manifests/00-crd-extension-console-link.yaml

```diff
@@ -53,4 +53,16 @@ spec:
             location:
               type: string
               description: Determines which dropdown menu the link is added to (HelpMenu, UserMenu)
```

Comment:
This comment needs to be updated as well.

Reply:
+1